### ISSUE TITLE
[Accessibility][Supporting the Platform- Property Pane_Ask a question (Bot Asks)]: At High contrast mode, their is no difference between selected and unselected tab at Bot asks, user input and other.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2135]
**User Experience** 
User with vision disability will not get to know what exactly control is selected if it will not properly visible after applying high contrast mode.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen at high contrast mode.
5. Observe the issue.

**Actual:** 
The low vision user will not be able to identify the difference between selected and unselected tab item.

**Expected:**
Layout of controls should be properly visible so that user will able to know the didderence between selected and unselected tab after applying high contrast black/white theme.

**MAS Impacted:** [MAS4.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B672c8a8b-ed4f-463f-868f-e8131caf9b74%7D)

**Attachment:**
![At high contrast mode the selected tab is not highlighted](https://user-images.githubusercontent.com/60913131/75563683-c1a6ea00-5a70-11ea-8f4c-76d37486fbf1.png)

**Comment by cwhitten** [comment]
@ashish315 can you properly label this issue? thanks!

**Comment by ashish315** [comment]
@cwhitten Here, the issue is, in normal mode the user will able to know which tab item is selected but in HCM their is no difference by which we can know which node is selected. Even in HCM, the low vision user might get confused as the controls are not look like tab item.

**Comment by ashish315** [comment]
@cwhitten The issue is labeled now, sorry for the delay.

**Comment by beyackle** [comment]
This might be blocked by a bug in Fabric. https://github.com/OfficeDev/office-ui-fabric-react/issues/12208

**Comment by Padmini171991** [comment]
@beyackle Will verify if the issue is repro/not repro in latest fabric controls and then confirm here.

**Comment by ashish315** [comment]
As checked with Fabric team, the issue also repro on the fabric home page. So, HCL is marking this as external. Will share the tracking bug ID soon.

**Comment by awalia13** [comment]
As this bug is dependent on Fabric Team so we will mark this bug as Fixed/External. And when Fabric issue will be fixed then we will again reach out to composer team to take the latest fabric changes and revalidate this bug again after code merge.

**Comment by ashish315** [comment]
The fabric team log the bug with bug ID [6407](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/6407)

**Comment by ashish315** [comment]
As checked on below environment the issue is still repro.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
Browser: Microsoft Edge Canary {Version 83.0.474.0 (Official build) canary (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
